apps/site/assets/ts/stop/components/DepartureList.tsx

```diff
@@ -28,40 +38,61 @@ const DepartureList = ({
   );
 
   let departures: DepartureInfo[] = [];
+  const groupedAlerts = alertsByRoute(alerts);
+  const alertsForRoute = groupedAlerts[route.id] || [];
 
+  const alertsByDirectionObj = alertsByDirectionId(alertsForRoute);
+  const alertsAffectingBothDirectionsArray = alertsAffectingBothDirections(
+    alertsForRoute
+  );
+
+  const alertsDirectionArray = alertsByDirectionObj[directionId]
+    ? alertsByDirectionObj[directionId]
+    : [];
+
+  const allAlerts = concat(
+    alertsAffectingBothDirectionsArray,
+    alertsDirectionArray
+  ).filter(alert => {
+    return ["detour", "suspension", "shuttle"].includes(alert.effect);
```

Comment:
suggestion (non-blocking) : It looks like detour, suspension, and shuttle are considered [here](https://github.com/mbta/dotcom/blob/master/apps/site/assets/ts/stop/components/DepartureTimes.tsx#L38) and in https://github.com/mbta/dotcom/pull/1609 as high-priority alerts. Maybe it would be helpful for consistency to define a function along the lines of `isHighPriorityAlert` that could be used in both of these places so that the logic for what alerts should be displayed is shared.